### ISSUE TITLE
Error occurs at the time of the flatten existing package.json jspm properties

### ISSUE THREAD

**Issue by MSakamaki** [jspm/jspm-cli#1627]
`Flatten existing package.json jspm properties?`

Problems can occur if you do not find the jspm to package.json at the time of this question

```sh
$ jspm init
Package.json file does not exist, create it? [Yes]: Yes
Init Mode (Quick, Standard, Custom) [Quick]: Custom
Prefix package.json properties under jspm? [Yes]: No
Flatten existing package.json jspm properties? [Yes]: Yes

err  TypeError: Cannot read property 'forEach' of undefined
    at PackageConfig.setPrefix (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/lib/config/package.js:199:19)
    at /Users/msakamaki/project/jspm/sample2/node_modules/jspm/lib/config/index.js:426:19
    at tryCatcher (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/node_modules/bluebird/js/release/promise.js:503:31)
    at Promise._settlePromise (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/node_modules/bluebird/js/release/promise.js:560:18)
    at Promise._settlePromise0 (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/node_modules/bluebird/js/release/promise.js:605:10)
    at Promise._settlePromises (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/node_modules/bluebird/js/release/promise.js:684:18)
    at Async._drainQueue (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/node_modules/bluebird/js/release/async.js:126:16)
    at Async._drainQueues (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/node_modules/bluebird/js/release/async.js:136:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/msakamaki/project/jspm/sample2/node_modules/jspm/node_modules/bluebird/js/release/async.js:16:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

I think trying to solve this problem. what to do if you can not find the jspm of package.json

 1. set to re-enter
 2. Forced to No

Branch is 0.17

**Comment by guybedford** [comment]
Can you share what the value is of your package.json when you get this error?

**Comment by guybedford** [comment]
Ah right I see what you mean. Thanks I've pushed a fix to the 0.17 branch.

**Comment by guybedford** [comment]
Fixed in jspm 0.17.0-beta.12.
